.idea/libraries/selenium_java_3_12_0.xml

```diff
@@ -0,0 +1,9 @@
+<component name="libraryTable">
```

Comment:
This file shouldn't be present in the master

Reply:
Added file to .gitignore